doc/subtransactions.rst

```diff
@@ -2,6 +2,10 @@
 Subtransactions in ZODB 3
 =========================
 
+.. caution::
```

Comment:
Good catch, this file is *definitely* out of date. Subtransactions were replaced with savepoints, and none of those function calls taking a parameter will work any more.

I think this could just be completely removed.

Reply:
Done.